recipe/build.sh

```diff
@@ -7,6 +7,14 @@ if [[ "${target_platform}" == "linux-ppc64le" ]]; then
   export CXXFLAGS="${CXXFLAGS//-fno-plt/}"
 fi
 
+IS_PYPY=$(${PYTHON} -c "import platform; print(int(platform.python_implementation() == 'PyPy'))")
+LIB_PYTHON="${PREFIX}/lib/libpython${PY_VER}${SHLIB_EXT}"
+if [[ ${IS_PYPY} == "1" ]]; then
+    INC_PYTHON="$PREFIX/include/pypy${PY_VER}"
+else
+    INC_PYTHON="$PREFIX/include/python${PY_VER}"
+fi
```

Comment:
Please drop the pypy bits, we're not going to do that here.

Reply:
Done in https://github.com/conda-forge/mlir-python-bindings-feedstock/pull/32/commits/0f43913ef8c470741cfb52d776a4be7485986430.